cli/exchange/service.go

```diff
@@ -476,14 +476,38 @@ func ServiceUpdatePolicy(org string, credToUse string, service string, jsonFileP
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, "Incorrect policy format in file %s: %v", jsonFilePath, err)
 	}
 
+	// Set default built in properties before publishing to the exchange
+	properties := policyFile.Properties
+	for i, property := range properties {
+		// remove the {"name":"prop1", "value":"value1"} from the generated service.policy.json template and set defualt built in properties
+		if property.Name == "prop1" {
+			properties = append(properties[:i], properties[i+1:]...)
+		}
```

Comment:
Please do not remove any user defined properties.

Reply:
👍 I will remove this block and keep whatever inside service.policy file